public/index.html

```diff
@@ -7,5 +7,388 @@
   </head>
   <body>
     <div id="root"></div>
+    <div class="first-page">
+      <header class="header">
+        <a href="#">
+          <img
+            class="logo header__logo"
+            src="./img/logo.svg"
+            alt="Site logo"
+          >
+        </a>
+        <nav class="header__nav nav">
+          <ul class="nav__list">
+            <li class="nav__item">
+              <a href="#" class="nav__link nav__link--active">Home</a>
+            </li>
+            <li class="nav__item">
+              <a href="#" class="nav__link">Phones</a>
+            </li>
+            <li class="nav__item">
+              <a href="#" class="nav__link">Tablets</a>
+            </li>
+            <li class="nav__item">
+              <a href="#" class="nav__link">Accessories</a>
+            </li>
+          </ul>
+        </nav>
+        <div class="header__buttons">
+          <a href="#" class="header__buttons--item">
+            <img
+              class="header__buttons--fav"
+              src="./img/fav.svg"
+              alt="Favourites"
+            >
+          </a>
+          <a href="#" class="header__buttons--item">
+            <img
+              class="header__buttons--cart"
+              src="./img/cart.svg"
+              alt="Cart"
+            >
+          </a>
+        </div>
+      </header>
+      <main class="main container">
+        <section class="promo section">
+          <div class="promo__main">
+            <div class="promo__scroll scroll scroll--left"></div>
+            <div class="promo__image">
+              <img
+                src="./img/Banner.svg"
+                class="promo__banner"
+                alt="Banner"
+              >
+            </div>
+            <div class="promo__scroll scroll scroll--right"></div>
+          </div>
+          <div class="promo__slide">
+            <div class="promo__line promo__line--active"></div>
+            <div class="promo__line"></div>
+            <div class="promo__line"></div>
+          </div>
+
+        </section>
+
+        <section class="hot section">
```

Comment:
what is hot?

Reply:
It was meant to be "hot-prices") But there is no actual need in this class, so I removed it.